docs/task-files.md

```diff
@@ -132,6 +133,24 @@ PolicyTemplate:
 
 The `!ReadFile` function will take 1 string argument, a file path, and return the contents of the file as a string.
 
+### !Cmd
+
+The `!Cmd` function will take 1 string argument, a shell command, and return the output from the
+shell command as a string.
+
+Example:
+
+```yaml
+Parameters:
+  Note: !Cmd 'echo "Deployed by `whoami` on `date`'
```

Comment:
i am not too fond of this example. it might seem nice, but in practice this will - every time org-formation runs - generate a new value for this parameter and therefore force the stack to update regardless of whether the actual stack changed (the parameter changed because `date` has a new value).

Reply:
changed to `Note: !Cmd 'echo "Deployed by `whoami`"'`